### PR TITLE
Fix import versions for GPTQ

### PR DESCRIPTION
**Pull request by jerryzh168**
Summary:
also exposed some additional quantizers from gptq.py

Test Plan:
CI

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: test/quantization/model.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from torch.nn import functional as F
 
 def prepare_inputs_for_model(inps):
+    inps = inps.squeeze(0)
```
Comment (cpuhrsch) [inline test/quantization/model.py:15]: Why is this needed?

Comment (jerryzh168) [reply test/quantization/model.py:15]: original input from lm-eval is 2d

**Review by cpuhrsch — APPROVED** [review]
As a follow up please send some docs and shape checks. Thank you!
